superx/routes/home.py

```diff
@@ -115,7 +115,12 @@ def add_item():
     cart_list.append(item)
     session['cart'] = cart_list
 
-    return ''
+    was_city_chosen = False
+    if 'city' in session:
+        was_city_chosen = True
```

Comment:
Is it possible that session["city"] == None or ""?
If so, you might want to use:
```
if session.get("city") 
# the default of get is None, if it doesn't exist.
```

You can also cut these 3 lines to:
`was_city_chosen = True if 'city' in session else False`